editor/app/crashes/[record_locator]/page.tsx

```diff
@@ -83,6 +83,9 @@ export default function CrashDetailsPage({
 
   return (
     <>
+      <title>{`${crash.cris_crash_id} - ${crash.address_primary ? crash.address_primary : ""} ${
+        crash.address_secondary ? "& " + crash.address_secondary : ""
+      }`}</title>
```

Comment:
This title looks great, but it looks like it's duplicating the `title` element in the DOM. Maybe using the [`next/head`](https://nextjs.org/docs/pages/api-reference/components/head)
component would replace instead of duplicate the title?

<img width="583" alt="Screenshot 2025-03-25 at 10 14 02 AM" src="https://github.com/user-attachments/assets/bc61fc1f-55bf-4dc9-a3af-6533a83c351c" />

Reply:
The next/head component [doesnt work](https://github.com/vercel/next.js/discussions/53529#discussioncomment-6628991) in the app router :thinking:

Reply:
I know having multiple `title` elements isn't valid, but I was wondering what happens if a page has more than one title element, and so far it seems like it affects SEO and browsers could potentially randomly choose which title to use

Reply:
I only did a quick bit of searching on this problem, but it seems like the only way to deal with this is with `useEffect`. Here's a little component the bot suggested

```javascript
export default function DynamicTitle({ title }) {
  useEffect(() => {
    // Update the document title using the browser API
    if (title) {
      document.title = title;
    }
  }, [title]);

  // This component doesn't render anything visible
  return null;
}
```

Reply:
ha this is what i found too and i just pushed up that change